app/src/main/java/com/guru/bluetooth/client/BluetoothClient.kt

```diff
@@ -0,0 +1,56 @@
+package com.guru.bluetooth.client
+
+import android.annotation.SuppressLint
+import android.bluetooth.BluetoothDevice
+import com.guru.bluetooth.BluetoothConnectionService
+import java.io.BufferedInputStream
+import java.io.File
+import java.io.FileInputStream
+import java.io.IOException
+import java.nio.ByteBuffer
+
+class BluetoothClient(device: BluetoothDevice) : Thread() {
+
+    @SuppressLint("MissingPermission")
```

Comment:
We should not suppress this warning, it might lead to security risks.
Instead, we should handle the permission here.